docs/wiki/EL/JSON-RPC.md

```diff
@@ -0,0 +1,113 @@
+# JSON-RPC
+
+---
+
+The JSON-RPC specification is a remote procedure call protocol encoded in JSON based on [OpenRPC](https://open-rpc.org/getting-started). It allows calling functions on a remote server, and for the return of the results.
+It is part of the Execution API specification which provides a set of methods to interact with the Ethereum blockchain.
+It is better known to be the way of how the clients interact with the network, even how the consensus layer (CL) and the execution layer (EL) interact.
+This section provides a description of the JSON-RPC methods.
+
+## JSON-RPC Methods
+
+There are a lot of JSON-RPC methods, but for summarizing purposes here are divided into the following prefix categories: *debug*, *engine* and *eth*, and all of them share a common structure:
+
+```json
+{
+  "id": 1,
+  "jsonrpc": "2.0",
+  "method": "<prefix_methodName>",
+  "params": [...]
+}
+```
+
+Where:
+- `id`: A unique identifier for the request.
+- `jsonrpc`: The version of the JSON-RPC protocol.
+- `method`: The method to be called.
+- `params`: The parameters for the method. It can be an empty array if the method does not require any parameters. Other ones may have default values if not provided.
+
+Here is a few list of the JSON-RPC methods:
+
+### Debug
+
+| **Method**               |      **Params**       | **Description**                                                 |
+|--------------------------|:---------------------:|-----------------------------------------------------------------|
+| debug_getBadBlocks       |  no mandatory params  | returns and array of recent bad blocks that the client has seen |
+| debug_getRawBlock        |     block_number      | returns an RLP-encoded block                                    |
+| debug_getRawHeader       |     block_number      | returns an RLP-encoded header                                   |
+| debug_getRawReceipts     |     block_number      | returns an array of EIP-2718 binary-encoded receipts            |
+| debug_getRawTransactions |        tx_hash        | returns an array of EIP-2718 binary-encoded transactions        |
+
+### Engine
```

Comment:
I would reorder these in more logical way rather than alphabetically. Start with eth, then debug and finish with engine api, highlight that it's meant for client to client communication, not user client